dotcom-rendering/src/components/FrontSection.tsx

```diff
@@ -488,8 +528,9 @@ export const FrontSection = ({
 	discussionApiUrl,
 	collectionBranding,
 	isTagPage = false,
+	hasNavigationButtons = false,
 }: Props) => {
-	const isToggleable = toggleable && !!sectionId && !containerLevel;
```

Comment:
We can now safely remove the !containerLevel check as we want to include show / hide toggles on containers with container levels